src/openforms/registrations/contrib/camunda/complex_variables.py

```diff
@@ -0,0 +1,187 @@
+"""
```

Comment:
I'm not entirely happy with this code, but it works (according to the tests) so I'll let it simmer for a bit and revisit at a later stage when we have to touch this again.